usaspending_api/common/csv_stream_s3_to_pg.py

```diff
@@ -0,0 +1,182 @@
+"""
+WARNING WARNING WARNING
+!!!!!!!!!!!!!!!!!!!!!!!
+This module must be managed very carefully and stay lean.
+
+The main functions: copy_csv_from_s3_to_pg and copy_csvs_from_s3_to_pg
+are used in distributed/parallel/multiprocess execution (by Spark) and is
+pickled via cloudpickle. As such it must not have any presumed setup code that would have run (like Django setup,
+logging configuration, etc.) and must encapsulate all of those dependencies (like logging config) on its own.
+
+Adding new imports to this module may inadvertenly introduce a dependency that can't be pickled.
+
+As it stands, even if new imports are added to the modules it already imports, it could lead to a problem.
+"""
```

Comment:
Just noting how fragile and worrisome this comment/description reads. That may be intentional and get the point across, just may be a TODO or tech debt to make this less fragile if this is the case.

Reply:
I think I got it as "stable" as I can. It's a matter of what other code gets latched on to this module or added to things it depends on. The unit tests that exercise this should blow up if any of those occurrences happen.